requirements/prod.txt

```diff
@@ -236,6 +233,9 @@ isodate==0.5.4 \
 jingo==0.9.0 \
     --hash=sha256:12a983286519303c6afd3da6f0a7e1859e69effb5fe46109424cdfca382bea56 \
     --hash=sha256:37de5435622d05763bc55d1a3d0a7edcf998ece9e6048bb470db431d1aa83c3a
+jingo-minify==0.6.0 \
+    --hash=md5:2c8c1a23a1c36bf4f4bd95150e75ecfb \
```

Comment:
That won't work, allowed hash algorithms for `--hash` are `sha256`, `sha384`, `sha512`.

We have a script that generates that for you, see http://addons-server.readthedocs.io/en/latest/topics/development/deps.html ; For just a single straightforward dependency like this without multiple packages you can just use:
`hashin jingo-minify==0.6.0 requirements/prod.txt`